psalm.xml

```diff
@@ -13,5 +13,11 @@
     </projectFiles>
 
     <issueHandlers>
+        <DeprecatedMethod>
+            <errorLevel type="suppress">
+                <!-- This issue needs to be resolved before upgrading to Lexer 3 -->
+                <referencedMethod name="Doctrine\Common\Lexer\Token::offsetGet"/>
```

Comment:
Which is the issue here? Can you reference to where in the code?

Reply:
In Lexer 1, tokens were associative arrays. Lexer 2 turns them into objects by introducing a new class `Token`. In order to make the migration easier, the new Token class implements `ArrayAccess` so that calls like this one are possible in both versions:

https://github.com/egulias/EmailValidator/blob/0170967656f55a87054fde439ef35a22df91468b/src/EmailLexer.php#L210

Avoiding the deprecation would mean, going from `$this->token['value']` to `$this->token->value` which is what #339 does already.